app/assets/stylesheets/components/page_specific/_floating_chat.scss

```diff
@@ -81,11 +89,246 @@
+  width: 100%;
+  margin: 0;
+  padding: .3em $select-padding;
+  padding-right: $select-padding + $arrow-width + $select-padding;
+  border: 1px solid $color-grey-light;
+  border-radius: .15em;
+  color: $color-grey-seven;
+  font-weight: 500;
+  background-color: white;
+  background-image: linear-gradient(45deg, transparent 50%, currentColor 50%), linear-gradient(-45deg, transparent 50%, currentColor 50%);
+  background-size: $arrow-width / 2 $arrow-width / 2;
+  background-position: calc(100% - #{$arrow-width / 2 + $select-padding}) center, calc(100% - #{$select-padding}) center;
+  background-repeat: no-repeat;
+  box-shadow: none;
+  transition: box-shadow 0.2s;
+
+  &:focus {
+    outline: 0;
+    box-shadow: 0 0 0 .15em hsla(0, 0%, 0%, .1);
+  }
+
+  &::-ms-expand {
+    display: none; // Strip IE10+ native arrow
+  }
+}
+
+//webchat buttons
+.mobile-webchat__button-container {
+  display: flex;
+  justify-content: space-between;
+
+  .mobile-webchat__button {
+    width: 48%;
+  }
+  // general button styles
+  .mobile-webchat__button--whatsapp,
+  .mobile-webchat__button--webchat {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: left;
+    font-weight: 700;
+    padding: 2px 5px;
+    transition: top 0.2s ease;
+
+    &:active {
+      box-shadow: 0px 1px $color-green-primary;;
+      top: 4px;
+    }
+
+    &:focus {
+      outline: 0;
+    }
+
+    svg {
+      width: 55px;
+      height: 45px;
+      padding: 4px;
+    }
+
+    .form-button__text {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: flex-start;
+
+      .form-button__text-heading {
+        display: block;
+        font-size: 0.8rem;
+        line-height: 0.9rem;
+      }
+
+      .form-button__text-service {
+        font-size: 0.85rem;
+        // min-width 345
+        @include respond-to((345 / $base-font-size) * 1em) {
+          font-size: 1rem;
+        }
+      }
+    }
+  }
+
+  .mobile-webchat__button--whatsapp {
+    background-color: #25D366; // whatsapp color
+    color: $color-white;
+    
+    &.is-hidden {
+      display: none;
+    }
+  }
+
+  .mobile-webchat__button--webchat {
+    background-color: $color-grey-pale;
+
+    &.disabled {
+      pointer-events: none;
+      background-color: $color-grey-normal;
+    }
+  }
+}
+
+// other contact methods link
+.mobile-webchat__contact {
+  margin-top: 10px;
+  text-align: right;
+
+  .mobile-webchat__contact--link {
+    font-weight: 700;
+    font-size: 0.8rem;
+  }
+}
+
+// popup close button
+.mobile-webchat__close {
+  font-size: 20px;
+  font-weight: 700;
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 30px;
+  height: 30px;
+  padding: 3px;
+  text-align: center;
+  color: $color-white;
+  border: none;
+  border-radius: 0 .15em 0 .15em;
+  background-color: $color-green-primary;
+  transition: color 0.25s, background-color 0.25s, opacity 0.4s linear 0.4s;
+  z-index: 91;
```

Comment:
This value will need looking at together with above - it is in front of the menu and grey overlay as it is.

Reply:
This issue relative to the menu and grey overlay has now been fixed